class  B4
{
	int test()
	{
		try
		{
			return 20;
		}
		catch(ArithmeticException ex)
		{
			
		}
		return 30;
	}
}
